internal/engine/pod.go

```diff
@@ -151,19 +149,11 @@ func maybeTrackPod(ms *store.ManifestState, action k8swatch.PodChangeAction) (*s
 		// CASE 2: We have a set of pods for this ancestor UID, but not this
 		// particular pod -- record it
 		podInfo = &store.Pod{
-			PodID:      podID,
-			StartedAt:  startedAt,
-			Status:     status,
-			Namespace:  ns,
-			HasSynclet: hasSynclet,
-			SpanID:     runtimelog.SpanIDForPod(podID),
-		}
-		if isCurrentDeploy {
-			// Only attach a new pod to the runtime state if it's from the current deploy;
-			// if it's from an old deploy/an old Tilt run, we don't want to be checking it
-			// for status etc.
-			runtime.Pods[podID] = podInfo
+			PodID: podID,
```

Comment:
these two cases are increasingly similar, is there any way to consolidate? (I have an instinct that yes but I'm not seeing exactly how, so 🤷‍♀)

Reply:
done